src/components/HeaderAuthOptions/HeaderSignUp/HeaderSignUp.js

```diff
@@ -74,11 +172,22 @@ class HeaderSignUp extends Component {
                 "aria-label": "Username",
               }}
             />
+            {submit && !usernameVal && <div className='image-div'><img
+                src={'red-x.svg'}
+                alt='red-x'
+                className={!usernameVal ? 'red-x' : 'green-check'}
+                onMouseEnter={!usernameVal ? () => this.onHover(this.state.openUser, 'openUser') : undefined}
```

Comment:
I'm using hover to display the error messages. It works well for the web browser, but we may want to implement an onClick also. For mobile it may be essential.